### ISSUE TITLE
Clarification could be useful

### ISSUE THREAD

**Issue by bartekus** [adobe/redux-saga-promise#3]
Since `redux-saga-promise` and [`redux-saga-thunk`](https://github.com/diegohaz/redux-saga-thunk) share similar intent while differing in implementations, perhaps it would be valuable for the community to have a bit of highlight/comparison between the two?

**Comment by ronen** [comment]
Yes, good idea.  Would you be willing to write a comparison as a PR?
